dev-python/twilio/twilio-6.29.0.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6} )
+inherit distutils-r1
+
+DESCRIPTION="Helper library for the Twilio API"
+HOMEPAGE="https://github.com/twilio/twilio-python http://www.twilio.com/docs/python/install"
+SRC_URI="https://github.com/twilio/${PN}-python/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86""
+IUSE="test"
+
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	dev-python/six[${PYTHON_USEDEP}]
```

Comment:
Are you convinced all those dependencies are needed at build time only? Because you haven't declared any runtime dependencies.